backend/.env.example

```
@@ -1,39 +1,31 @@
-MAIL_PASS=
-
-# API Documentation
-SWAGGER_PATH=/api/docs
-SWAGGER_TITLE=StellarSplit API
-SWAGGER_DESCRIPTION=API for StellarSplit - Split bills instantly with crypto
-SWAGGER_VERSION=1.0.0
+
+
+# ── API Versioning ────────────────────────────
+# Default API version used in URI prefix /api/v{API_VERSION}/
+API_VERSION=1
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)